src/controllers/message.js

```diff
@@ -0,0 +1,20 @@
+const { StatusCodes } = require('http-status-codes')
+const { model, STATES } = require('mongoose')
+const Book = require('../../models/Book')
+const User = require('../../models/User')
+const { BadRequestError, NotFoundError } = require('../../errors')
+const Message = require('../../models/Message')
+
+//Create message
+const createMessage = async (req, res) => {
+    const { userId } = req.user
+    const { recievedByUser, messageContent } = req.body
+    const message = await Message.create({
+        postedByUser: userId,
+        recievedByUser,
```

Comment:
Nice work Manizha! The message schema looks good, just there's a small typo in **receivedByUser** . I tested this message router on Postman and I think we need to define receivedByUser : userId when we create a message, otherwise, it will return an error since it's required by the body.

Reply:
Thanks Lara for your review, ( userId ) is an Id which we take it from auth token so it is equal to one who post the message, that is why I am taking it from req.user, and the recievedByUser id is the id which you put it into body. so when you send the request you should write the id of user you want to send the message!

example

{
    "recievedByUser":"641cc0a509842e20b0bb987f",
    "messageContent":"Hey, Is someone borrowed your book, I am interested!"
}

Reply:
You are right Manizha! I realized an error that I got is because I tested in Postman with **receivedByUser** meanwhile you wrote it as **recievedByUser.**

Reply:
@jpremo could you please approve this PR? Thank you.